### DISCUSSION TITLE
Has anyone gotten errors debugging Aspire in VS Code on Mac after upgrading to Sequoia?

### DISCUSSION THREAD

**Discussion by jeffa00** [microsoft/aspire#5822] [Q&A]
After upgrading to Sequoia I get an error trying to start debugging on the AppHost project.

The specific error I get is "The JSON-RPC connection with the remote party was lost before the request could complete."

Google and Bing haven't yielded much help. I find a similar error in Visual Studio from back in 2019ish, but nothing recent.

I've uninstalled extensions, then VS Code, but no joy.

I can debug the web app just fine, but the error always occurs when trying to start AppHost.

I'm trying to figure out if it is indeed something in Sequoia. I have a work mac with Sonoma that doesn't have the issue.

Just curious if anyone else has seen this or if I'm just super lucky. :-)

**Comment by StefanW2015** [comment]
Having the same issue after upgrading to Sequoia. Also tried a brand new starter application from the latest aspire workload with no success. 

Debugging Web, Console or MAUI applications (After disabling Hot-Reload) is running fine. 

Perhaps you should file this as an issue to get more attention?!

**Reply by jeffa00** [reply]
I hate to file an issue until I see that at least one other person has the same problem. I don't want to file an issue that turns out to just be something misconfigured on my system. I'll convert this to an issue now. Thanks for the response.

**Comment by Waste-Treatment** [comment]
Using the following:

1. Sequoia 15.0
2. Intel Core I9
3. VS Code
4. Set and hit break point in AppHost -> Program.cs

Of course, wasn't able to hit any break points in the other projects but I think that is expected.?.?

**Reply by jeffa00** [reply]
I wonder if the Intel vs Apple silicon could be a factor? I'm on Apple silicon.

**Reply by Waste-Treatment** [reply]
Which generation of Apple Silicon do you have? I wonder if there is a difference between M1 and M3?

**Reply by StefanW2015** [reply]
I have an M3 Pro and I'm having the issue

**Reply by jeffa00** [reply]
I am on M2

**Reply by Fiontan** [reply]
same issue in m3 and similar/same issue mentioned elsewhere : https://github.com/dotnet/aspire/issues/5860
